cmd/ko/README.md

```diff
@@ -181,6 +189,33 @@ to whatever `kubectl` context is active.
 `ko delete` simply passes through to `kubectl delete`. It is exposed purely out
 of convenience for cleaning up resources created through `ko apply`.
 
+
+### With `minikube`
+
+You can use `ko` with `minikube` via a Docker Registry, but this involves
+publishing images only to pull them back down to your machine again.  To avoid
+this `ko` exposes `--local` or `-L` options to instead publish the images to
+the local machine's Docker daemon.
+
+This would look something like:
+
+```shell
+# Use the minikube docker daemon.
+eval $(minikube docker-env)
+
+# Make sure minikube is the current kubectl context.
+kubectl config use-context minikube
+
+# Deploy to minikube w/o registry.
+ko apply -L -f config/
+```
+
+A caveat of this approach is that it will not work if your container is
+configured with `imagePullPolicy: Always` because despite having the image
+locally, a pull is performed to ensure we have the latest version.
```

Comment:
`ko` doesn't emit digests for `-L` because the digest computation is ambiguous when running through the `daemon.Write` path.  We *tag* it with the digest we think it should have, which avoids the annoying K8s default that `:latest` is `imagePullPolicy: Always`.  I've expanded to be more precise :P